### PR TITLE
Replaced slang terms

### PR DESCRIPTION
**Pull request by JeremyTBradshaw**
There is no such thing as the "Exchange Online admin" role.  There is no place in these docs for writers' comfort-zone slang.  Readers came here for a reason, we shouldn't steer them wrong with terminology that is not going to be found anywhere within the products that they've come here to learn about.

Also, updated the recommendation to also assign the "Service Support Administrator" (not "Service admin") role so that it is clearly just that, a recommendation.  It's not mandatory to do that and it's subjective.

### CODE REVIEW

File: microsoft-365/admin/add-users/about-exchange-online-admin-role.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ ms.assetid: 097ae285-c4af-4319-9770-e2559d66e4c8
 description: "Exchange Online admins manage your organization email and mailboxes and, for example, recover deleted items in a user's mailbox."
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/about-exchange-online-admin-role.md]:
```suggestion
description: "Exchange administrators manage your organization's email and mailboxes and, for example, recover deleted items in a user's mailbox."
```

File: microsoft-365/admin/add-users/about-exchange-online-admin-role.md [hunk 2]
```diff
@@ -25,13 +25,13 @@ ms.assetid: 097ae285-c4af-4319-9770-e2559d66e4c8
...
+To help you administer Microsoft 365, you can [assign](assign-admin-roles.md) users permissions to manage your organization's email and mailboxes from the <a href="https://go.microsoft.com/fwlink/p/?linkid=2059104" target="_blank">Exchange admin center</a>. You do this by assigning them to the Exchange Administrator role.
   
- **Tip**: When you assign someone to the Exchange admin role, also assign them to the Service admin role. This way they can see important information in the Microsoft 365 admin center, such as the health of the Exchange Online service, and change and release notifications.
+ **Tip**: When you assign someone to the Exchange Administrator role, it is recommended to assign them to the Service Support Administrator role. This way they can see important information in the Microsoft 365 admin center, such as the health of the Exchange Online service, and change and release notifications.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/about-exchange-online-admin-role.md]:
```suggestion
 **Tip**: When you assign someone to the Exchange Administrator role, we recommend assigning them to the Service Support Administrator role. This way they can see important information in the Microsoft 365 admin center, such as the health of the Exchange Online service, and change and release notifications.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JeremyTBradshaw, kindly commit the minor additional suggested changes. Many thanks.

**Comment by JeremyTBradshaw** [comment]
@JohanFreelancer9 I think I've done so correctly.  Here's hoping.  Just let me know if I need to redo or do anything more.  Thanks!

File: microsoft-365/admin/add-users/about-exchange-online-admin-role.md [hunk 3]
```diff
@@ -22,16 +22,16 @@ search.appverid:
...
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/about-exchange-online-admin-role.md]:
```suggestion
> [!TIP]
> When you assign someone to the Exchange Administrator role, we recommend assigning them to the Service Support Administrator role. This way they can see important information in the Microsoft 365 admin center, such as the health of the Exchange Online service, and change and release notifications.
```

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @JeremyTBradshaw.

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Pull request merged**